include/FAudio.h

```diff
@@ -1062,7 +1062,7 @@ FAUDIOAPI void FAudioVoice_GetOutputMatrix(
 );
 
 /* Removes this voice from the audio graph and frees memory. */
-FAUDIOAPI void FAudioVoice_DestroyVoice(FAudioVoice *voice);
+FAUDIOAPI uint32_t FAudioVoice_DestroyVoice(FAudioVoice *voice);
```

Comment:
I think this counts as an ABI break, so we may have to export a new function that includes the return value :(